colabs/huggingface/Huggingface_wandb.ipynb

```diff
@@ -199,12 +191,39 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## Model Artifacts\n",
     "\n",
-    "### 📈 Track key information effortlessly by default\n",
+    "You will also be able to view your model checkpoints in the W&B [Artifacts](https://docs.wandb.ai/guides/artifacts) page of the run above. These artifacts store all the metadata associated with this model, the W&B runs consuming it, and the whole lineage of upstream and downstream artifacts!\n",
+    "\n",
+    "Here's an example of the artifacts generated from the above run\n",
```

Comment:
Change text to

> This image shows a summary of all the checkpoints (artifact) generated from the above run. 

We should embed the image rather than link to gdrive, but ideally can we just link to this view (if project is public) so people can see output summary in the UI?